modules/nf-core/fgbio/zipperbams/main.nf

```diff
@@ -0,0 +1,57 @@
+process FGBIO_ZIPPERBAMS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::fgbio=2.0.2" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fgbio:2.0.2--hdfd78af_0' :
+        'quay.io/biocontainers/fgbio:2.0.2--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(ubam)
+    tuple val(meta), path(mbam)
```

Comment:
I think `mapped_bam` and `unmapped_bam` are a bit more clear here

Reply:
agree! modified thanks